compiler/damlc/lib/DA/Cli/Damlc/Packaging.hs

```diff
@@ -147,6 +151,8 @@ createProjectPackageDb projectRoot (disableScenarioService -> opts) modulePrefix
         let (pkgNode, pkgId) = vertexToNode vertex in
         -- stable packages are mapped to the current version of daml-prim/daml-stdlib
         -- so we don’t need to generate interface files for them.
+
+        -- ???
         unless (pkgId `Set.member` stablePkgIds || pkgId `Set.member` dependenciesInPkgDbIds) $ do
```

Comment:
Did I understand correctly that this needs to become something like
```haskell
  case depType of
    StablePkg -> pure ()
    RegularDep -> <copy files and recachePkgDb>
    DataDep -> <the big do here>
```
?

Reply:
yep that sounds about right.